### PR TITLE
Fix team users download and add some unit tests

### PR DESCRIPTION
**Pull request by BradHN1**
#367 User downloads broken

### CODE REVIEW

**Comment by BradHN1** [comment]
Fix two other issues in this PR:
#357 - contact cadmin - send e-mail just to primary contact not all cadmins
#374 - problem running locally

File: src/_main_/settings.py [hunk 1]
```diff
@@ -18,6 +18,12 @@
 import sentry_sdk
 from sentry_sdk.integrations.django import DjangoIntegration
 
+
+DJANGO_ENV = os.environ.get("DJANGO_ENV","remote")
+DEV_SERVER = False
```
Comment (Opoku-Agyemang) [inline src/_main_/settings.py:23]: can we use a name like `RUN_SERVER_LOCALLY`
Also can we add comments to describe what each of these stand for?

File: src/_main_/settings.py [hunk 2]
```diff
@@ -63,6 +70,7 @@
     'massenergize-canary-api.us-east-2.elasticbeanstalk.com',
     'massenergize.test',
     'massenergize.test:3000',
+    'communities.massenergize.test'
```
Comment (Opoku-Agyemang) [inline src/_main_/settings.py:73]: massenergize.test covers this OR you could also do *.massenergize.test

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]
I think we are mixing several things in one PR which makes it hard to review and isolate things.  Happy to +1 but its risky

File: src/_main_/settings.py (hunk 2 above)
Comment (BradHN1) [reply src/_main_/settings.py:73]: ok

File: src/_main_/settings.py (hunk 1 above)
Comment (BradHN1) [reply src/_main_/settings.py:23]: Yes, done

**Comment by BradHN1** [comment]
You're right, my bad.
Most of them are pretty simple fixes.

**Review by Opoku-Agyemang — APPROVED** [review]

**Pull request merged**